src/components/Auth/UsersCard/table/UsersTable.tsx

```diff
@@ -169,6 +172,7 @@ export const UsersTable: React.FC<React.PropsWithChildren<UsersTableProps>> = ({
           </DataTableBody>
         </DataTableContent>
       </DataTable>
+      {shouldShowNukedUsers && <UsersCleared />}
```

Comment:
nit: let's remove the nuked terminology for "cleared" or "clearedAll" for clarity

Reply:
For sure! I also found the term "nuked" kind of gauche, but kept it to follow with the action definitions in `src/store/auth/actions.tsx`. I'll change the names to `cleared`!